home/home_2.1/HelloWorld.cpp

```diff
@@ -0,0 +1,92 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <unordered_map>
+#include <sstream>
+
+const std::string FILENAME = "user_stats.txt";
+
+std::unordered_map<std::string, int> readStats() {
+    std::unordered_map<std::string, int> stats;
+    std::ifstream infile(FILENAME);
+    std::string line;
+    while (std::getline(infile, line)) {
+        std::istringstream iss(line);
+        std::string name;
+        int count;
+        if (iss >> name >> count) {
+            stats[name] = count;
+        }
+    }
+    return stats;
+}
+
+void writeStats(const std::unordered_map<std::string, int>& stats) {
+    std::ofstream outfile(FILENAME, std::ios::trunc);
+    for (const auto& pair : stats) {
+        outfile << pair.first << " " << pair.second << "\n";
+    }
+}
+
+void deleteStats() {
+    std::ofstream outfile(FILENAME, std::ios::trunc);
+}
+
+void displayUsage() {
+    std::cerr << "Usage: <name> or <name> delete or bread\n";
+}
+
+int main() {
+    std::unordered_map<std::string, int> stats = readStats();
+    
+    while (true) {
```

Comment:
this main loop seems to be too long - consider encapsulation of some part to functions or maybe some business logic can be a part of encapsulated data structure from the comment above